### PR TITLE
Update sources.js

### PR DESCRIPTION
**Pull request by musthemag**
Switched Opinion to List.

### CODE REVIEW

File: server/config/sources.js [hunk 1]
```diff
@@ -16,8 +16,8 @@ const sources = {
 		idV1: 'NTlhNzEyMzMtZjBjZi00Y2U1LTg0ODUtZWVjNmEyYmU1NzQ2-QnJhbmRz'
 	},
 	opinion: {
-		sectionsId: 'MTE2-U2VjdGlvbnM=',
-		uuid: 'ec66fcc8-cd25-11de-a748-00144feabdc0'
+		type: 'list',
```
Comment (matthew-andrews) [inline server/config/sources.js]: Ah I think we need to keep the sections id - I suspect that's how it gets linked to the opinion stream. @andygout can confirm!

Comment (andygout) [inline server/config/sources.js]: Correct:

screen shot 2016-05-12 at 09 01 41

**Comment by andygout** [comment]
I wonder if we need to change the two mentions of 'page' to 'list' here as well? Am having a look.

https://github.com/Financial-Times/next-graphql-api/blob/master/server/lib/schema.js#L64-L70

**Comment by andygout** [comment]
^^^ Seems that if the `uuid` now relates to a list rather than a page, that it should be `capi.list()` that calls it as an argument. `editorsPicks` (also a list in `server/config/sources.js`) has the same logic in `server/lib/schema.js`.

**Comment by andygout** [comment]
@matthew-andrews Changes made with Mus - please would you review and 👍  if all okay?

**Comment by matthew-andrews** [comment]
looks ok +1, although tests are failing

**Comment by andygout** [comment]
Thanks - as soon as they are fixed.

**Pull request merged**